sdk/communication/azure-communication-phonenumbers/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## 1.1.0-beta.15 (Unreleased)
 
 ### Features Added
+- API version `2022-12-01` is now the default for Phone Numbers clients.
```

Comment:
This one is not necessary; it was already reflected in 1.1.0-beta.13 below.

Reply:
Removed.